### ISSUE TITLE
Dedicated pages that show available artists and albums

### ISSUE THREAD

**Issue by eldar** [geoffrey45/alice-core#15]
I really like the UI of your player and I wanted to suggest a few minor changes to make it a bit more intuitive. I only have a small local collection of my favorite albums and upon adding the root folder to the library I would expect to see the albums on the home page of the app. The only way to find them currently is to use the search by album or artist. It took me a while to figure this out. Ideally, there would to be a page and a corresponding shortcut on the left panel listing artists and another page listing albums. The search text box could be displayed in the more prominent position, that is above the main pain and not far to the right. Also, there needs to be a button to rescan the library -- at the moment I have to restart the server after I add an album under the root directory.

**Comment by cwilvx** [comment]
Hello @eldar Thank you for opening this issue.

I totally agree with the idea of having pages to browse artists and albums. I didn't have these pages planned for the next release, but I agree they should be there. I was wanted to implement the stats feature first (tracking listening activity), to make the album and artist pages more resourceful. ie. by having information like most played, most recent, etc.

However, I will prepare simple album and artist pages for the next release. 

About the search input on the far right, it's placed there to act as quick search. Instead of going to the search page, you just start typing to search, wherever you are. But, in the next release, there is a "top results" feature that will be added to the search. This will make the quick search panel quite heavy. So the quick search panel will be moving from the sidebar to a dialog. One that can be triggered by using `Ctrl + P`.

About the rescan button. The server actually triggers a function to scan the added directory. It just takes a few seconds to load all the tracks in the new dir. [If the added dir appears empty in the folder page, then the scan is underway]. I havent't figured out a way to show the scan progress on the client given that the app is based on the client-server architecture. The progress is only shown in the terminal. I'll check out whether it's working as expected, if not, I'll fix it.

**TL:DR**
The artists and albums pages have been schedule for next release.

**Comment by cwilvx** [comment]
@eldar 

I'm the only person working on the design of the app, and I can't say I'm good at it. If you can figure out any more ways to improve the UI/UX of the app, don't hesitate to open an issue. You can check out the current figma designs of the app at:

https://www.figma.com/file/OOnZMcNZZER0QrWeK2UV44/Alice?node-id=0%3A1&t=iGwt35OPvRyj6bof-1

[the design has not been updated for a while, but it has not changed that much. I'll find time to update it]
Feel free to fork the design and tweak it to your liking. I'll be very happy to take a look and maybe improve a thing or two.

Thank you.

**Comment by cwilvx** [comment]
Fixed in previous releases.
